csm/conf/post_install.py

```diff
@@ -67,8 +72,13 @@ async def execute(self, command):
         if Conf.get(const.CONSUMER_INDEX, 'systemd>csm>csm_agent>restart_on_failure') == 'true':
             self._configure_system_auto_restart()
         self._configure_service_user()
-        self._configure_rsyslog()
+        #TODO: Add check for rsyslog host and port in confstore
+        self._set_syslog_details()
+        if self._is_syslog:
+            self._configure_rsyslog()
         self._allow_access_to_pvt_ports()
+        if not self._replacement_node_flag:
+            self.create()
```

Comment:
Remove node replacement related code for now... we still not designed the node replacement scenario.. We will take this up later